cmd/transcriber/config/config.go

```diff
@@ -118,10 +118,10 @@ func (cfg CallTranscriberConfig) IsValid() error {
 		return fmt.Errorf("CallID parsing failed")
 	}
 
-	if cfg.PostID == "" {
-		return fmt.Errorf("PostID cannot be empty")
-	} else if !idRE.MatchString(cfg.PostID) {
-		return fmt.Errorf("PostID parsing failed")
+	if cfg.TranscriptionID == "" {
+		return fmt.Errorf("TranscriptionID cannot be empty")
+	} else if !idRE.MatchString(cfg.TranscriptionID) {
+		return fmt.Errorf("TranscriptionID parsing failed")
```

Comment:
Do we really need to have these checks in here? If they are wrong reporting the failure will fail anyway so not sure what we are gaining. They just seemed misplaced.

Reply:
True, the report will fail anyway. But I thought at least this way we will get the error as a failure to validate the URL in the docker logs... But I guess we would get that from the plugin's own validation check. So nevermind, putting it back.